src/modules/webhooks/get-interaction/1.0/module.json

```diff
@@ -1,40 +1,1053 @@
+    },
+    {
+      "name": "Source Details Person Phone Person Id",
+      "value": "$.interaction.sourceDetails.person.phone.personId",
+      "defaultValue": "sourceDetailsPersonPhonePersonId"
+    },
+    {
+      "name": "Source Details Person Phone Session Id",
+      "value": "$.interaction.sourceDetails.person.phone.sessionId",
+      "defaultValue": "sourceDetailsPersonPhoneSessionId"
+    },
+    {
+      "name": "Source Details Person Phone Phone Number",
+      "value": "$.interaction.sourceDetails.person.phone.phoneNumber",
+      "defaultValue": "sourceDetailsPersonPhonePhoneNumber"
+    },
+    {
+      "name": "Source Details Person Device Id",
+      "value": "$.interaction.sourceDetails.person.deviceId",
+      "defaultValue": "sourceDetailsPersonDeviceId"
+    },
+    {
+      "name": "Source Details Person Person Id",
+      "value": "$.interaction.sourceDetails.person.personId",
+      "defaultValue": "sourceDetailsPersonPersonId"
+    },
+    {
+      "name": "Source Details Person Session Id",
+      "value": "$.interaction.sourceDetails.person.sessionId",
+      "defaultValue": "sourceDetailsPersonSessionId"
+    },
+    {
+      "name": "Source Details Person Phone Number",
+      "value": "$.interaction.sourceDetails.person.phoneNumber",
+      "defaultValue": "sourceDetailsPersonPhoneNumber"
+    },
+    {
+      "name": "Source Details Phone Number",
+      "value": "$.interaction.sourceDetails.phoneNumber",
+      "defaultValue": "sourceDetailsPhoneNumber"
+    },
+    {
+      "name": "Source Details Phone Number Formatted Type",
+      "value": "$.interaction.sourceDetails.phoneNumberFormatted.type",
+      "defaultValue": "sourceDetailsPhoneNumberFormattedType"
+    },
+    {
+      "name": "Source Details Phone Number Formatted Number Type",
+      "value": "$.interaction.sourceDetails.phoneNumberFormatted.numberType",
+      "defaultValue": "sourceDetailsPhoneNumberFormattedNumberType"
+    },
+    {
+      "name": "Source Details Phone Number Formatted Country Calling Code",
+      "value": "$.interaction.sourceDetails.phoneNumberFormatted.countryCallingCode",
+      "defaultValue": "sourceDetailsPhoneNumberFormattedCountryCallingCode"
+    },
+    {
+      "name": "Source Details Phone Number Formatted National Number",
+      "value": "$.interaction.sourceDetails.phoneNumberFormatted.nationalNumber",
+      "defaultValue": "sourceDetailsPhoneNumberFormattedNationalNumber"
+    },
+    {
+      "name": "Source Details Phone Number Formatted Number",
+      "value": "$.interaction.sourceDetails.phoneNumberFormatted.number",
+      "defaultValue": "sourceDetailsPhoneNumberFormattedNumber"
+    },
+    {
+      "name": "Source Details Phone Number Formatted Country",
+      "value": "$.interaction.sourceDetails.phoneNumberFormatted.country",
+      "defaultValue": "sourceDetailsPhoneNumberFormattedCountry"
+    },
+    {
+      "name": "Source Details Phone Number Formatted Number Parts Area Code",
```

Comment:
Lets remove the `Number Parts` from the name and defaultValue for these as well. So for areaCode, exchange, and lastFour